tcn_hpl/data/components/PTG_dataset.py

```diff
@@ -107,10 +107,26 @@ def __getitem__(self, idx):
 
         :return: features, targets, and mask of the window
         """
+        
+        # print(f"size of dataset: {self.__len__()}")
```

Comment:
An option other than commenting out lots of `print` or logging code is to add a verbosity parameter to the constructor of this object that can be toggled via a config file instead of code changes.